### ISSUE TITLE
How to run JJ2 on Mac

### ISSUE THREAD

**Issue by Jones-S** [deathkiller/jazz2-native#50]
I downloaded the game here: https://www.gog.com/de/game/jazz_jackrabbit_2_collection
and I am trying to run it on mac and I tried to follow the procedure described here:
https://deat.tk/jazz2/help

I copied the files from the folder downloaded from GOG into the given Source directory, however after starting up the resurrection app I can't do anything.
Could somebody tell me if these files look alright, or if I copied the wrong ones?

I copied the content of x86 (also tried x64) to the indicated `Source` directory.  But on resurrection startup nothing changes...

<img width="441" alt="image" src="https://github.com/deathkiller/jazz2-native/assets/7059580/fd370400-5528-450c-8818-27ec830cbc66">

Thanks for any hints.

**Comment by deathkiller** [comment]
It looks like you downloaded Windows version. You need to download macOS version (https://github.com/deathkiller/jazz2/releases/download/2.5.0/Jazz2_2.5.0_MacOS.zip) and install ARM64 (if you have new Mac) or x64 (if you have old Mac) version. Then run the game, you can then press `Enter` in main menu to open `Source` directory in Finder, then copy original files there. And then you can play the game.

**Comment by Jones-S** [comment]
Hm no, but this is the download of the Resurrection App, isn't it? Of course I downloaded the Mac version for that, but I want to use the Resurrection App to play the windows version of JJ2 (because a mac version was never released). 
I wonder which the `original files` are you talk about.
I thought the original files were the source files of the game I downloaded on gog (which are shown in the screenshot).
Or do I completely misunderstand?

**Comment by deathkiller** [comment]
Files shown in your screenshot are from Windows version of Jazz² Resurrection (not files from GOG). You've probably got your files mixed up.
And yes, you need to place original Jazz Jackrabbit 2 files (the Windows version from GOG) to `Source` directory. The directory should contain `Anims.j2a`, `Data.j2d` and a lot of `*.j2l` and `*.j2t` files (which is not the one in your screenshot).

**Comment by Jones-S** [comment]
Oh alright, my bad. I tried this together not at my own computer and requested the screenshot. Could very well be that things got mixed up. Let me try again...

**Comment by Jones-S** [comment]
Ah but If I download that I get only the file `GOG_Galaxy_Jazz_Jackrabbit_2_The_Secret_Files.exe`...

**Comment by deathkiller** [comment]
You need to extract files from the GOG installer you downloaded. It's written also here - https://deat.tk/jazz2/help#which-versions-of-jazz-jackrabbit-2-game-are-supported:

> If you are not able to install/extract original files from GOG.com installer, you can use **innoextract**.

Innoextract works fine on Windows, but I don't know how it works on Mac. Unfortunately, I don't use Mac, so I have no idea how to extract it there.

**Comment by Jones-S** [comment]
Oh alright, that's the missing piece then. I read about that innoextract, but it was not really clear to me how to use it and so on. I will try. Thank you that helps
